spec/index.bs

```diff
@@ -5414,6 +5414,48 @@ dictionary GPUImageCopyTexture {
 
 Issue(gpuweb/gpuweb#69): Define the copies with {{GPUTextureDimension/1d}} and {{GPUTextureDimension/3d}} textures.
 
+### <dfn dictionary>GPUImageCopyTextureTagged</dfn> ### {#gpu-image-copy-texture-tagged}
+
+WebGPU textures hold raw numeric data, and are not tagged with semantic metadata describing colors.
+However, {{GPUQueue/copyExternalImageToTexture()}} copies from sources that describe colors.
+
+A {{GPUImageCopyTextureTagged}} is a {{GPUImageCopyTexture}} which is additionally tagged with
+color space/encoding and alpha-premultiplication metadata, so that semantic color data may be
+preserved during copies.
+This metadata only affects the semantics of the {{GPUQueue/copyExternalImageToTexture()}} call.
+
+<script type=idl>
+dictionary GPUImageCopyTextureTagged : GPUImageCopyTexture {
+    GPUPredefinedColorSpace colorSpace = "srgb";
+    boolean premultipliedAlpha = false;
+};
+</script>
+
+<dl dfn-type=dict-member dfn-for=GPUImageCopyTextureTagged>
+    : <dfn>colorSpace</dfn>
+    ::
+        Describes the color space and encoding used to encode data into the destination texture.
```

Comment:
Forgive me, I know this functionality has been discussed endlessly but I'm still not clear on something: WebGPU textures already have both linear and sRGB format variants that dictate how they're sampled, right? (`rgba8unorm` vs. `rgba8unorm-srgb`) Why isn't that sufficient to determine how the color values should be copied?